### PR TITLE
Integrate arm with the whole system; Rework robot description and launch files; Improve documentation

### PR DESCRIPTION
**Pull request by kapi117**

### CODE REVIEW

**Comment by rayferric** [comment]
Please compress `docs/gs_dark_autonomy.png`. Ideally, any media files in our repo should stay under a few hundred KiB, unless it would significantly hurt quality.

**Comment by rayferric** [comment]
I don't think that cyclone config should stay in `kalman_arm_config`. Generally, we want this repo to be pure ROS code that does not depend on any kind of DDS. For PC and GS we use cyclone config from kalman_ws, which - importantly - is a separate deployment repo. Maybe we should keep that config locally on the RPi? Or rather, we should solve DDS issues once for all and have a good common config in kalman_ws for every deployment... It reminds me that I still must test out Zenoh, whose config steps include setting cyclone to localhost only mode, so it would potentially play nicely with the cyclonedds.xml in kalman_arm_config, since it also specifies localhost only networking. If everything else is not viable, we could introduce this config into kalman_ws and choose it based on detected system architecture.

File: kalman_bringup/README.md [hunk 1]
```diff
@@ -1,7 +1,54 @@
...
+It allows us to quickly create launch files with custom configurations while maintaining their readability.
+
+**Example Compass Calibration launch:**
+
+```python
+from kalman_bringup import *
+
+def generate_launch_description():
+    return gen_launch({
+        "hardware": {
```
[the remark this answers is not in the document]
Comment (kapi117) [reply kalman_bringup/README.md:41]: done

File: kalman_master/README.md [hunk 2]
```diff
@@ -1,28 +1,36 @@
-# kalman_base
+# Master
 
-This package contains nodes to interface between master device on serial port and ROS2. Additionally, it provides abstract drivers for that communication. It also contains configuration files and launch files for the base.
+driver for our custom "Master" device
```
[the remark this answers is not in the document]
Comment (rayferric) [reply kalman_master/README.md:3]: Added a whole paragraph

**Pull request merged**